### ISSUE TITLE
Check and update log libraries

### ISSUE THREAD

**Issue by labra** [weso/rdfshape-api#276]
Check if the libraries employed are affected by the log4j vulnerability

**Comment by ulitol97** [comment]
Regarding the Log4Shell exploit for Log4j, we are not affected by it. Our logging stack is:
- Frontend: [SLF4J](https://www.slf4j.org/) via a wrapper ([scala-logging](https://index.scala-lang.org/lightbend/scala-logging/scala-logging/3.9.2))
- Backend: [Logback](https://logback.qos.ch/)

SLF4J is a simple facade for logs with no vulnerabilities associated to Log4Shell, further information is provided [here](https://www.slf4j.org/log4shell.html):

> The SLF4J API is just an API which lets message data go through.

> Logback does NOT offer a lookup mechanism at the message level. Thus, it is deemed safe with respect to CVE-2021-44228.

On a further note, the Logback team have recently [discovered vulnerabilities](https://jira.qos.ch/browse/LOGBACK-1591) that can grant RCE but require previous write access to the server and have been patched in the latest version.

**Summing up: we are safe from Log4Shell and I'll update the Logback version in future commits for further security.**
